### DISCUSSION TITLE
Multilingual Training

### DISCUSSION THREAD

**Discussion by willfant** [coqui-ai/TTS#1547] [General Q&A]
Hey guys, I already searched but didn't find anything. It is possible to train a voice in a language and synthesize in another language?

My goal to train a model with my voice and then use it for different languages.

I saw this demo in the coqui website so I suppose it is possible.

Is there any documentation on that?

Thanks!

**Comment by neurlang** [comment]
Hello I'm new here, so correct me if I'm wrong.
But the models on the site each speak their own language trained on, just with a different (i.e. yours) person's style.
So no, you can't train your speech model and then magically speak other language it was not trained on, unless I'm missing something and I would like to know about that.

Thinking about it, if you train on IPA, then you can speak many languages in your dataset and get a merged (multilingual) model, but last time I tried it, I got bad results because I had Russian in there and the model said x (ch) at the begging of every word.

I think each language have it's own rules to place (move) the gaps in between the phones, and the number of gaps might be equal or related to number of words spoken. I've already developed a model to propagate the gaps in various languages, but its difficult to prove this theory right/wrong because of lack of data. It might be impossible to merge tonal languages like Chinese with other languages.

I'm still interested to know if this is possible or other's opinions.

**Reply by erogol** [reply]
Your first point is correct.

IPA works for similar languages like German and Luxemburgish, but I don't think it would be more general.
